functions.R

```diff
@@ -564,18 +564,20 @@ get_rejected_lab_data <- function(raw_lab_result_data,
   if(nrow(rejected_lab_data) > 0){
 
     #  uf_id found but names do not match
-    uf_id_match_no_name_match <- rejected_lab_data %>%
+    uf_id_match_no_dob_name_match <- rejected_lab_data %>%
       inner_join(all_rows_in_redcap_results %>%
                    filter(is.na(redcap_repeat_instrument)),
                  by = c("uf_id" = "record_id"),
                  suffix = c("_personhub", "_rcresults")) %>%
-      select(uf_id, name_in_person_hub_data = display_name)
+      select(uf_id,
+             birth_date, dob_in_person_hub_data = date_of_birth,
+             patient = patient_personhub, name_in_person_hub_data = display_name)
 
     rejected_lab_data_with_rejection_reason <- rejected_lab_data %>%
+      select(-c(patient, birth_date)) %>%
```

Comment:
```suggestion
      select(uf_id,
             dob_in_person_hub_data = date_of_birth,
             name_in_person_hub_data = display_name)

    rejected_lab_data_with_rejection_reason <- rejected_lab_data %>%
```

I think you want `patient` and `birth_date` from the `rejected_lab_data` rather than `uf_id_match_no_dob_name_match`

Reply:
I choose both so that the epic peeps can see where the difference lie in dob and name. If the uf id matches and it appears in rejected_lab_data that means that both the dob and name is different between lab result file and person hub data.

Reply:
but if the UF-IDs don't match you get neither.